### PR TITLE
Social

### PR DESCRIPTION
**Pull request by logophobia**
Fix footer and home page social media

### CODE REVIEW

File: CollAction/Views/Home/Index.cshtml [hunk 1]
```diff
@@ -138,11 +138,11 @@
...
     <div id="project-share">
         <div class="container">
-            <div class="row">
+            <div class="row panel panel-default">
                 <div class="col-xs-12" id="share-title">
-                    <h3>Share Coll Action</h3>
+                    <h3>Spread it further</h3>
                 </div>
-                <div class="col-xs-12 col-md-3 col-md-offset-5">
+                <div class="col-xs-12 col-md-3 col-md-offset-5" style="margin-bottom: 40px">
```
Comment (minayao) [inline CollAction/Views/Home/Index.cshtml]: Maybe move the style to home.scss

File: CollAction/Views/Shared/_Footer.cshtml [hunk 2]
```diff
@@ -43,11 +43,9 @@
         <h5>@Localizer["Follow Us"]</h5>
         <ul class="social-list">
           <li><a target="_blank" href="https://www.facebook.com/sharer/sharer.php?u=http://collaction.org/"><i class="fa fa-facebook marginright"></i></a></li>
-          <li>
-            <a target="_blank" href="https://twitter.com/intent/tweet?original_referer=http://collaction.org/&amp;url=http://collaction.org/"><i class="fa fa-twitter marginright"></i></a>
-          </li>
-          <li><a target="_blank" href="http://www.pinterest.com/pin/create/button/?url=http%3A%2F%2Fcollaction.org%2F"><i class="fa fa-pinterest marginright"></i></a></li>
-          <li><a target="_blank" href="https://plus.google.com/share?url=http://collaction.org/"><i class="fa fa-google-plus marginright"></i></a></li>
+          <li><a target="_blank" class="twitter-follow-button" href="https://twitter.com/CollAction_org"><i class="fa fa-twitter marginright"></i></a></li>
```
Comment (minayao) [inline CollAction/Views/Shared/_Footer.cshtml]: Are you also fixing these links on the homepage (in the 'Spread it further' box)? I'm still getting the facebook error there. Or is that a different task?

File: CollAction/Views/Home/Index.cshtml (hunk 1 above)
Comment (logophobia) [inline CollAction/Views/Home/Index.cshtml]: Sure

File: CollAction/Views/Shared/_Footer.cshtml (hunk 2 above)
Comment (logophobia) [inline CollAction/Views/Shared/_Footer.cshtml]: I ... can't. Don't have Mac to test on.

Comment (minayao) [inline CollAction/Views/Shared/_Footer.cshtml]: That's ok, go ahead and merge for now.

**Pull request merged**